eeauditor/auditors/aws/Amazon_EC2_Auditor.py

```diff
@@ -788,4 +789,421 @@ def ec2_serial_console_access_check(cache: dict, awsAccountId: str, awsRegion: s
             "Workflow": {"Status": "RESOLVED"},
             "RecordState": "ARCHIVED"
         }
-        yield finding
\ No newline at end of file
+        yield finding
+
+
+@registry.register_check("ec2")
+def ec2_ami_age_check(cache: dict, awsAccountId: str, awsRegion: str, awsPartition: str) -> dict:
+    """[EC2.5] EC2 Instances should use AMIs that are less than 6 months old"""
```

Comment:
Can we switch this one to 3 months? Figure most companies doing this seriously are baking golden images every 2 or 4 weeks, and may have a N - 2 version policy

Reply:
Absolutely, I'm all for putting it lower!  Lowered it to 3 months :)